dipy/reconst/sfm.py

```diff
@@ -420,7 +420,24 @@ def design_matrix(self):
         return sfm_design_matrix(self.gtab, self.sphere, self.response,
                                  'signal')
 
-    def fit(self, data, mask=None):
+    def fit_solver2voxels(self, isopredict, vox_data, vox, parallel=False):
+        # In voxels in which S0 is 0, we just want to keep the
+        # parameters at all-zeros, and avoid nasty sklearn errors:
+        if not (np.any(~np.isfinite(vox_data)) or np.all(
+                vox_data == 0)):
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
+
+        if parallel is True:
+            return {vox: self.solver.fit(
+                self.design_matrix, vox_data - isopredict[vox]
+            ).coef_}
+        else:
+            return self.solver.fit(
+                self.design_matrix, vox_data - isopredict[vox]
+            ).coef_
+
+    def fit(self, data, mask=None, n_threads=1):
```

Comment:
threads or processes?  our conventional names are `num_processes` or `num_threads`

Reply:
Ideally this would be `num_threads` then, thanks for the clarification. Ultimately we'll probably want to switch the joblib backend to loky/cloudpickle once joblib figures out how to play nice with pickle5 compat...

Reply:
I just see tht you are using `backend='multiprocessing'`  so it should be `num_processes`. The multiprocessing module does not use threads. It spawns multi processes.